README.md

```diff
@@ -0,0 +1,35 @@
+## React Tutorial from Scratch– A Step-by-Step Guide (2021 Update)
+
+This repo holds all the code for a React todos application. You can follow the step by step [guide to building the awesome project here](https://ibaslogic.com/react-tutorial-for-beginners/).
+
+## [Click to See the Todos App](https://ibaslogic.github.io/react-todo-project/)
+
+The code is available in the `src` folder. In there, you’ll see two folders named `classBased` and `functionBased` holding the class component and function component code respectively.
+
+### Clone the project
+
+You can download or clone this project by running this command from your terminal:
+
+```
+git clone https://github.com/silviatofana/react-todo-app
+```
+
+This will create a directory in the name of the project folder.
+
+Once you have the project files and folders bootstrapped, open it with your text editor.
+
+Next, switch inside the project directory and run:
+
+```
+npm install
+```
+
+This will install all the necessary dependencies in the local `node_modules` folder.
+
+Finally, start your development server by running:
+
+```
+npm start
+```
+
```

Comment:
- [x] Please kindly add steps author,Technologies used, it will make your README look professional